plugins/plugins.cpp

```diff
@@ -30,6 +30,13 @@
 // 8Mode
 #include "8Mode/src/8mode.hpp"
 
+// Aaron Static
+#include "AaronStatic/src/plugin.hpp"
+// Nasty override of this required function for now:
+NVGcolor prepareDisplay(NVGcontext *vg, Rect *box, int fontSize) {
```

Comment:
this is not needed since you are including the plugin.cpp

Reply:
hmm, if I remove this the build fails on:
```
/usr/bin/ld: ../../plugins/plugins.a(plugin.cpp.o):(.bss.pluginInstance__AaronStatic+0x0): multiple definition of `pluginInstance__AaronStatic'; ../../plugins/plugins.a(plugins.cpp.o):(.bss.pluginInstance__AaronStatic+0x0): first defined here
```

and I haven't been able to resolve it ..

Reply:
just use extern for this plugin instance then